src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -362,6 +736,27 @@ public CollectionList listCollections() throws PineconeException {
         return collections;
     }
 
+    /**
+     * Deletes a collection.
+     * <p>
+     * Deleting a collection is an irreversible operation. All data in the collection will be lost.
+     * This method tells Pinecone you would like to delete a collection, but it takes a few moments to complete the operation.
+     * Use the describeCollection() method to confirm that the collection has been deleted.
```

Comment:
Does deleting a collection also delete the index in the collection?

Reply:
Nope